lib/MusicBrainz/Server/Entity/CollectionItem.pm

```diff
@@ -0,0 +1,51 @@
+package MusicBrainz::Server::Entity::CollectionItem;
+use Moose;
+use namespace::autoclean;
+
+use DateTime::Format::Pg;
+
+use MusicBrainz::Server::Entity::Types;
+use MusicBrainz::Server::Types qw( PgDateStr );
+
+has 'entity' => (
+    is => 'rw',
+    isa => 'Entity',
+    required => 1,
+);
+
+has 'added' => (
+    is => 'rw',
+    isa => 'PgDateStr',
```

Comment:
```suggestion
    isa => 'Maybe[Str]',
```

(Moose doesn't know how to deal with `Maybe[PgDateStr]` for some reason, but it's fine to use `Str` in any case since we're not coercing from `DateTime` objects.)